### ISSUE TITLE
RefNeRF Real Dataset

### ISSUE THREAD

**Issue by grgkopanas** [google-research/multinerf#18]
Hi,

Thank you all for the great work and the release of the code. I want to try the RefNeRF in real datasets, starting by reproducing the results in the scenes from the paper. I noticed that, first, the only config file for RefNeRF is for the shiny blender dataset, and second that the scene files for the real scenes from the RefNeRF website don't match the structure of the other scenes usually created for NeRF (transforms.json etc are missing).

Is there an easy way to run the real scenes and do I need special config parameters to reproduce the results of the paper?

Best,
Georgios Kopanas

**Comment by StarsTesla** [comment]
Same issue here

**Comment by gkouros** [comment]
The codebase supports different dataset formats. The shiny dataset is of type _blender_ and the real dataset is of type _LLFF_. To train on real scenes with RefNeRF, you can use _config/blender_refnerf.gin_ but change _Config.dataset_loader_ from "blender" to "llff".

**Comment by gkouros** [comment]
Yeah, I had the same issue with one of my datasets. I think you need to 
disable Config.compute_normal_metrics.

**Comment by prakashknaikade** [comment]
How the shiny blender dataset is created in blender from this [blend files](https://dorverbin.github.io/refnerf/data/shiny_blender_source.zip) ?? @gkouros @grgkopanas 
I am new to blender.

**Comment by gkouros** [comment]
> How the shiny blender dataset is created in blender from this [blend files](https://dorverbin.github.io/refnerf/data/shiny_blender_source.zip) ?? @gkouros @grgkopanas I am new to blender.

The question is not relevant to this issue so create a new issue. In the meantime, you can look at similar issues in the original nerf repo e.g. https://github.com/bmild/nerf/issues/78.

**Comment by prakashknaikade** [comment]
https://github.com/google-research/multinerf/issues/136
